### PR TITLE
fix: nav and sidebar fix

### PR DESCRIPTION
**Pull request by sks-21**
- Fixed **navbar** and **sidebar**
- Updated navigation links and footer links
- added installation, typography and colours page
- refactored card and navigation component

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -343,25 +389,27 @@
...
 }
 
-.pd-5 {
+.pd-32 {
   padding: 32px;
 }
+
 /* || Imagery */
 
 img,
```
Comment (MaruthiPothuganti) [inline styles.css:415]: This style scan be moved to reset file.

File: styles.css [hunk 2]
```diff
@@ -343,25 +389,27 @@
...
 }
 
-.pd-3 {
+.pd-16 {
   padding: 16px;
 }
 
-.pd-4 {
+.pd-24 {
   padding: 24px;
```
Comment (MaruthiPothuganti) [inline styles.css:406]: can use rems instead of pixels

File: components/navigation/navigation.css [hunk 3]
```diff
@@ -55,6 +65,59 @@
...
+
+.nav-search-bar {
+  width: 100%;
+  background-color: inherit;
+
+}
+
+.nav-search-bar:focus {
+  border: none;
+  outline: none;
```
Comment (MaruthiPothuganti) [inline components/navigation/navigation.css:86]: Outline should not be made none.

> It provides visual feedback for links that have "focus" when navigating a web document using the TAB key (or equivalent). This is especially useful for folks who can't use a mouse or have a visual impairment. If you remove the outline you are making your site inaccessible for these people

File: components/input/input.html [hunk 4]
```diff
@@ -2,180 +2,179 @@
...
+          <span class="form-icon"><i class="fa fa-user icon "></i></span>
+          <input type="text" id="std-input" class=" form-input form-input-text " placeholder="Enter username">
+        </div>
+        <div class="form-comp-1 flx-row flx-center">
+          <span class="form-icon"><i class="fa fa-envelope icon"></i></span>
+          <input type="text" id="std-input" class=" form-input form-input-text " placeholder="Enter email">
+        </div>
+        <div class="form-comp-1 flx-row flx-center">
+          <span class="form-icon"><i class="fa fa-key icon"></i></span>
+          <input type="text" id="std-input" class=" form-input form-input-pswd " placeholder="Enter password">
```
Comment (MaruthiPothuganti) [inline components/input/input.html:132]: type can be changed into password

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested couple of changes. Everything else looks good

File: colorspage/colorspage.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+        <!-- || Side Menu starts   -->
+        <aside class="menu flx-col flx-main-center">
+            <ul class="menu-list">
+                <li class="menu-list-title">Getting Started</li>
+                <li class="menu-list-item-active">
+                    <a href="#">Installation</a>
+                </li>
+                <li>
+                    <a href="/typography/typography.html">Typography</a>
```
Comment (kotalwar-krishna) [inline colorspage/colorspage.html:44]: can you use ./ at start please try this better practice according to me

File: components/navigation/navigation.css (hunk 3 above)
Comment (sks-21) [reply components/navigation/navigation.css:86]: Thanks for the info!

File: styles.css (hunk 2 above)
Comment (sks-21) [reply styles.css:406]: It's my style guide basically for consistent units across all screen sizes.

**Review by kotalwar-krishna — APPROVED** [review]
Good to merge

File: colorspage/colorspage.html (hunk 5 above)
Comment (sks-21) [reply colorspage/colorspage.html:44]: Ok

**Review by aryanandarnav13 — APPROVED** [review]
Good to go

**Pull request merged**